client/README.md

```diff
@@ -7,7 +7,7 @@
 - Switch to using Node 12: `nvm use 12`
 - From this directory, `npm run setup` to install front end dependencies and create an `.env` file
 - get a [Mapbox](https://account.mapbox.com/auth/signin/) API token and add that to your `.env` file as the `MAPBOX_TOKEN`
-- get the `API_URL` from a team member and add that to your `.env` file. Alternatively, you can [bring up](https://github.com/hackforla/311-data/blob/dev/docs/server_setup.md) your own local server and use that.
+- get the `API_URL` from a team member and add that to your `.env` file. Alternatively, you can [bring up](https://github.com/hackforla/311-data/blob/dev/docs/server_setup.md) your own local server and use that (if you are running your own local server, set API_URL to 'http://localhost:5000')
```

Comment:
nit: \`API_URL\` instead of API_URL for consistency

Reply:
ok, changed API_URL to `API_URL` for consistency